op-batcher/flags/flags.go

```diff
@@ -26,7 +26,7 @@ var (
 	// Required flags
 	L1EthRpcFlag = &cli.StringFlag{
```

Comment:
A string slice flag would be better than manually parsing strings.

Reply:
Yes I will try to modify this

Reply:
resolved